unit/solvers/refinement/string_constraint_instantiation/instantiate_not_contains.cpp

```diff
@@ -11,9 +11,13 @@
 #include <numeric>
 #include <java_bytecode/java_bytecode_language.h>
 #include <java_bytecode/java_types.h>
+
 #include <langapi/mode.h>
+#include <langapi/language_util.h>
```

Comment:
Why is this additional include necessary when there are no changes to the code?

Reply:
solvers/refinement/string_refinement.h no longer includes langapi/language_util.h

Reply:
Ah, ok!